README.md

```diff
@@ -72,6 +74,35 @@ Source this environment in and provide the password when prompted:
 $ source .env
 ```
 
+## Jupyter notebooks
+Though Jupyter notebooks and lab are well suited to running Visier connector code, the use of OS-level environment variables may not be ideal for many users in this setting. For this purpose and since version `0.9.9`, the connector supports [dotenv](https://pypi.org/project/python-dotenv/) to facilitate a more dynamic switching of Visier authentication parameters.
+
+### Jupyter OAuth2 example
+In order to authenticate using OAuth, you will need a registered client application configuration in your tenant. Administrator-level privileges are required in order to register OAuth clients. 
+
+Create an environment file to store the authentication parameters. If the file is called `.env`, the Python package `dotenv` will attempt to load that. If given an arbitrary name, that named has to be provided when loading the environment using `dotenv`. 
```

Comment:
After the OAuth client is registered in Visier, create an environment file to store the authentication parameters. If the file is called `.env`, the Python package `dotenv` attempts to load the file. If the file has a different name, you must provide that file name when loading the environment with `dotenv`.